### DISCUSSION TITLE
Issue lesson 6 with command " forge compile"

### DISCUSSION THREAD

**Discussion by AxelRoffi** [Cyfrin/foundry-full-course-cu#269] [General]
I have been following the tutorial and when I run "forge compile" I have this error

<img width="695" alt="Screenshot 2023-07-06 at 16 20 06" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/21120237/927078fa-de4b-41cc-9e77-8a7b265cac36">

I tried AI to debug but was not able to find a solution. 
If you have any idea, Thank you!!

**Comment by IAm0xAxe** [comment]
Hey ser @AxelRoffi, 
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/33568293/183445aa-88ea-4722-9685-34c82e23a4b6)
**You can use this:** `import "lib/forge-std/src/Script.sol";`
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/33568293/2de050ea-317a-4035-bb78-020cb266db88)

**Reply by AxelRoffi** [reply]
Hello, I have the file "lib/forge-std/src/Script.sol".
I imported the file as you said.
I still have the same error.
Thank you very much for trying!

**Comment by shikhar229169** [comment]
Can you upload your code on github and provide the repo link.

**Reply by AxelRoffi** [reply]
Here the link, I did not include script folder which is empty.
[https://github.com/AxelRoffi/foundry-simple-storage](url)
once again, thank you!!

**Reply by shikhar229169** [reply]
Actually ```forge build``` and ```forge compile``` both works for me on the repo you provided.
Also you don't need to add ```import "lib/forge-std/src/Script.sol"``` in your ```SimpleStorage.sol```.

Actually I have never worked in macOS, so I am not familiar with the dyld symbol error.

**Reply by shikhar229169** [reply]
Also consider to ask this problem here: https://t.me/foundry_support

**Reply by AxelRoffi** [reply]
thanks for trying!! have a nice day!!

**Comment by IAm0xAxe** [comment]
Can you send your code? `DeploySimpleStorage.s.sol` and `SimpleStorage.sol`

**Comment by AxelRoffi** [comment]
Hello, so this is an issue with Apple M1 and MacOS 11
There is a thread solving this is issue:
[https://github.com/foundry-rs/foundry/issues/2712]
I did not had the time to test it but it seems to work.
I would let you know.
best

**Reply by IAm0xAxe** [reply]
OKay Ser,Good luck!
